frontend/src/components/common/ReleaseNotes/ReleaseNotesModal.tsx

```diff
@@ -7,7 +7,7 @@ import { useTranslation } from 'react-i18next';
 import ReactMarkdown from 'react-markdown';
 
 export interface ReleaseNotesModalProps {
-  releaseNotes?: string;
```

Comment:
I don't think this commit is doing what the title says it's doing. But the title kind of hints to the problem I experienced in the screenshot I shared (if the text is white on white background -> screenshot makes sense).

Reply:
updated